webapp/src/components/Help.tsx

```diff
@@ -14,58 +16,84 @@ import { Stack, Container, useTheme } from '@mui/material'
 const Help = () => {
   const theme = useTheme()
   return (
-    <Container maxWidth={false} id="help" sx={{ textAlign: 'left', backgroundColor: theme.palette.background.default }}>
+    <Container
+      maxWidth={false}
+      id="help"
+      sx={{ textAlign: 'left', backgroundColor: theme.palette.background.default }}
+    >
       <Stack spacing={2}>
         <h2>Welcome to the {EnvironmentVars.DOT_NAME} CV Manager Website</h2>
         <p>
           This application shows the physical location and message counts for each RSU installed by the Colorado
           Department of Transportation at various road sites throughout Colorado.
         </p>
+
+        <p>
+          On the map interface, RSUs are represented by colored dots to indicate their current status. A green dot
+          means the RSU is online and actively reporting data. A yellow dot indicates the RSU is currently offline but
+          was recently active. A red dot shows the RSU is offline and has not reported in some time.
+        </p>
+        <BorderedImage src={status} alt="RSU Status Indicators on Map" />
+
         <p>
-          The map on this website will represent the location of each RSU with a red, green, or yellow dot (shown
-          below). A green dot represents an RSU that is online. A yellow dot represents an RSU that is offline but was
-          recently active. A red dot represents an RSU that is not currently online.
+          Selecting an RSU on the map will open a popup window along with a side configuration menu. The popup displays
+          detailed information including the RSU’s IP address, online status, last online time, milepost number, serial
+          number, and the number of messages it has reported. The configuration menu allows users, depending on their
+          access level, to perform actions such as retrieving the current message forwarding settings, modifying
+          configurations, checking firmware updates, applying firmware updates, or rebooting the RSU.
         </p>
-        <BorderedImage src={status} alt="RSU Statuses Shown on Map" />
+        <BorderedImage src={popup} alt="RSU Popup and Configuration Panel" />
+
         <p>
-          Clicking on the RSU will give the following information: IP address, online status, time last online, milepost
-          number, serial number, and number of message counts (shown below).
+          On the right side of the interface, there are two menu toggles that open menus with more detailed information
+          about the RSUs. The first is the RSU Status Menu, which shows a full list of all RSUs and their current status.
+          This menu can be toggled on or off by clicking the red X in the corner or by selecting the "Display RSU Status"
+          toggle again. From this menu, users also have the option to print a complete report or an error-specific report
+          for further analysis or record-keeping.
```

Comment:
I would like to see a few more features covered here:
1. Click on Location to select and center map on RSU and show detailed RSU status
2. Description of detailed/individual RSU status display